pkgs/top-level/all-packages.nix

```diff
@@ -5357,7 +5357,7 @@ with pkgs;
 
   inherit (ocamlPackages) haxe;
 
-  hxcpp = callPackage ../development/compilers/haxe/hxcpp.nix { };
```

Comment:
I would either add a `hxcpp` to rename to `rename.nix` or add an alias, as removing it breaks compatibility.